Ospdo qa comments changes (#815)
openstack-k8s-operators/data-plane-adoption: docs_user/modules/proc_preparing-controller-nodes-for-director-operator-adoption.adoc

export CONTROLLER_SSH="oc rsh -n $OSPDO_NAMESPACE -c openstackclient openstackclient ssh controller-0.ctlplane"

export $CONTROLLER1_SSH="oc -n $OSPDO_NAMESPACE rsh -c openstackclient openstackclient ssh controller-0.ctlplane"
export $CONTROLLER2_SSH="oc -n $OSPDO_NAMESPACE rsh -c openstackclient openstackclient ssh controller-1.ctlplane"

> export $CONTROLLER3_SSH="oc -n $OSPDO_NAMESPACE rsh -c openstackclient openstackclient ssh controller-2.ctlplane"

Reviewer: From the review call -- this looks unintended too.

$CONTROLLER1_SSH sudo pcs cluster stop
$CONTROLLER1_SSH sudo pcs cluster start
----

> .. Check if only controller-0 is started and the other 2 controllers are stopped:

Reviewer 2: ```suggestion
.. Check that only Controller-0 is started and that the other 2 Controllers are stopped:
+
```

Reviewer 2: Please commit this suggestion.

Author: ok

----
$CONTROLLER_SSH sudo pcs status
----

> .. Check if the openstack control plane is still operational: (May need to wait few minutes for the control plane to get operational. The control plane response will get slow after this point).
> ----
> $OS_CLIENT openstack compute service list
> ----

Reviewer 2: ```suggestion
.. Check if the {OpenStackShort} control plane is still operational: 
+
----
$OS_CLIENT openstack compute service list
----
+
[NOTE]
You might need to wait a few minutes for the control plane to get operational. The control plane response slows down after this point.
```
Why does the response slow down?

Reviewer 2: Please commit this suggestion. The step order is wrong in the preview.

Author: ok, 
about the question as I understand , the responce is slowing down, because the control plane services get downscaled from 3 to 1 , so the downscale takes time for the pacemaker services to sync on the 1 controller

> .. Once pacemaker is only managing one of the controllers, we can delete 2 of the controller VMs:
> Annotate controller-1 and controller-2 VMs for deletion:
> ----
> oc -n openstack annotate vm controller-1 osp-director.openstack.org/delete-host=true
> oc -n openstack annotate vm controller-2 osp-director.openstack.org/delete-host=true
> ----

Reviewer 2: ```suggestion
.. When Pacemaker is only managing one of the Controllers, delete 2 of the Controller VMs. The following example specifies Controller-1 and Controller-2 VMs for deletion:
+
----
$ oc -n openstack annotate vm controller-1 osp-director.openstack.org/delete-host=true
$ oc -n openstack annotate vm controller-2 osp-director.openstack.org/delete-host=true
----
```

Reviewer 2: Please commit this suggestion.

Author: ok

> .. Reduce the roleCount for controller role in the OpenStackControlPlane CR to "1":
> ----
> oc -n openstack patch OpenStackControlPlane overcloud --type json -p '[{"op": "replace", "path":"/spec/virtualMachineRoles/controller/roleCount", "value": 1}]'
> .. Ensure that the openstackclient pod is running on the same OCP nodes as the remaining controller VM. If its not on the same node, then move it by cordoning off the two nodes that have been freed up for RHOSO and deleting the openstackclient pod so that it gets rescheduled on the OCP node that has the remaining controller VM. Once the pod has moved to the correct node, uncordon all nodes.
> ----
> oc adm cordon $OSP18_NODE1
> oc adm cordon $OSP18_NODE2
> oc delete pod openstackclient
> oc adm uncordon $OSP18_NODE1
> oc adm uncordon $OSP18_NODE2
> ----

Reviewer 2: ```suggestion
.. Reduce the `roleCount` for the Controller role in the `OpenStackControlPlane` CR to "1":
+
----
$ oc -n openstack patch OpenStackControlPlane overcloud --type json -p '[{"op": "replace", "path":"/spec/virtualMachineRoles/controller/roleCount", "value": 1}]'
.. Ensure that the `OpenStackClient` pod is running on the same {OpenShiftShort} nodes as the remaining Controller VM. If the `OpenStackClient` pod is not on the same node, then move it by cordoning off the two nodes that have been freed up for {rhos_acro}. Then you delete the `OpenStackClient` pod so that it gets rescheduled on the {OpenShiftShort} node that has the remaining Controller VM. After the pod is moved to the correct node, uncordon all the nodes:
+
----
$ oc adm cordon $OSP18_NODE1
$ oc adm cordon $OSP18_NODE2
$ oc delete pod openstackclient
$ oc adm uncordon $OSP18_NODE1
$ oc adm uncordon $OSP18_NODE2
----
```

Author: ok

$CONTROLLER1_SSH sudo pcs cluster stop
$CONTROLLER1_SSH sudo pcs cluster start
----

.. Check if only controller-0 is started and the other 2 controllers are stopped:
----
$CONTROLLER_SSH sudo pcs status
----
.. Check if the openstack control plane is still operational: (May need to wait few minutes for the control plane to get operational. The control plane response will get slow after this point).
----
$OS_CLIENT openstack compute service list
----
.. Once pacemaker is only managing one of the controllers, we can delete 2 of the controller VMs:
Annotate controller-1 and controller-2 VMs for deletion:
----
oc -n openstack annotate vm controller-1 osp-director.openstack.org/delete-host=true
oc -n openstack annotate vm controller-2 osp-director.openstack.org/delete-host=true
----

> .. Reduce the `roleCount` for the Controller role in the `OpenStackControlPlane` CR to "1":

Reviewer 2: ```suggestion
.. Reduce the `roleCount` for the Controller role in the `OpenStackControlPlane` CR to `1`:
```

Author: ok
